build.gradle

```diff
@@ -14,7 +14,7 @@ buildscript {
     ext.versions = [
             "appCompat"         : "1.1.0",
             "coreTest"          : "2.0.0",
-            "coreKtx"           : "1.2.0",
+            "coreKtx"           : "1.5.0-alpha04",
```

Comment:
Can you briefly explain why `core` is upgraded to an alpha version?

Reply:
We need this to have access to the `WindowCompat.setDecorFitsSystemWindows(...)`.